tests/integration/conftest.py

```diff
@@ -83,12 +83,12 @@ async def application_fixture(
         timeout=20 * 60,
         idle_period=30,
     )
-
     # slow down update-status so that it doesn't intervene currently running tests
-    async with ops_test.fast_forward(fast_interval="5h"):
-        yield application
-
-    await model.remove_application(application.name, force=True, block_until_done=True)
+    # `with ops_test.fast_forward` is not used here since juju cleanup will cause the tests to
```

Comment:
I'm not following this, it seems that the next line uses `with ops_test.fast_forward`?

Reply:
This is the current line under test, this will either be finalized with `with ops_test.fast_forward` of `model.set_config`, whichever one fails the tests less.

Reply:
I've decided to stick with `op_test.fast_forward`, the comments have been updated accordingly.
```
# slow down update-status so that it doesn't intervene currently running tests
    # don't yield inside the context since juju cleanup is not reliable.
    # model.set_config(...) also doesn't work as well as the following code.
    async with ops_test.fast_forward(fast_interval="5h", slow_interval="5h"):
        pass
    yield application
```